api/routes/issues.js

```diff
@@ -0,0 +1,185 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+
+const Issue = require('../models/issue');
+const Comment = require('../models/comment');
+
+// GET / - list all created issues
+router.get('/', (req, res, next) => {
+    Issue.find()
+        .select('-__v')
+        .exec()
+        .then(docs => {
+            const response = {
+                count: docs.length,
+                issues: docs.map(doc => {
+                    return {
+                        description: doc.description,
+                        status: doc.status,
+                        _id: docs._id,
+                        request: {
+                            type: 'GET',
+                            url: 'http://localhost:3000/issues/' + doc._id
```

Comment:
This url shouldn't be hard coded. What happens if you deploy this to some other service and `localhost` becomes invalid?